{"uid":"13a7c03141831667","name":"test_changing_password[)TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB-)TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB-success]","fullName":"tests.wallet_tests.test_wallet_change_password#test_changing_password","historyId":"33f7fe9b44221c63713f58ac556cfd07","time":{"start":1732272403152,"stop":1732272403152,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55f289d6031a <unknown>\n#1 0x55f2898766e0 <unknown>\n#2 0x55f2898addc8 <unknown>\n#3 0x55f2898a9926 <unknown>\n#4 0x55f2898f56f6 <unknown>\n#5 0x55f2898f4d46 <unknown>\n#6 0x55f2898e9203 <unknown>\n#7 0x55f2898b7cc0 <unknown>\n#8 0x55f2898b8c9e <unknown>\n#9 0x55f289d2dd0b <unknown>\n#10 0x55f289d31c92 <unknown>\n#11 0x55f289d1ab3c <unknown>\n#12 0x55f289d32807 <unknown>\n#13 0x55f289d000df <unknown>\n#14 0x55f289d4f578 <unknown>\n#15 0x55f289d4f740 <unknown>\n#16 0x55f289d5f196 <unknown>\n#17 0x7f0a855b8609 start_thread","statusTrace":"request = <SubRequest 'driver' for <Function test_changing_password[)TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB-)TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB-success]>>\n\n    @pytest.fixture(autouse=True, scope=\"function\")\n    def driver(request):\n        options = webdriver.ChromeOptions()\n    \n        # Отключение анимаций\n        options.add_argument(\"--disable-animations\")\n    \n        prefs = {\"profile.managed_default_content_settings.images\": 2}  # 2 отключает загрузку изображений\n        options.add_experimental_option(\"prefs\", prefs)\n    \n        # Прочие настройки\n        options.add_argument(\"--no-sandbox\")\n        options.add_argument(\"--window-size=1280,720\")\n        options.add_argument('--disable-dev-shm-usage')\n        options.add_argument('--disable-gpu')\n    \n        # Путь к расширению\n        # project_path = os.path.dirname(os.path.abspath(__file__))\n        # extension_path = f\"{project_path}/extension/dist/chrome\"\n    \n        # CI\n        extension_path = \"/app/extension\"\n    \n        options.add_argument(f\"--load-extension={extension_path}\")\n    \n        # Инициализация драйвера\n>       driver = webdriver.Chrome(options=options)\n\n../conftest.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chromium/webdriver.py:66: in __init__\n    super().__init__(command_executor=executor, options=options)\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py:212: in __init__\n    self.start_session(capabilities)\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py:299: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fb18028dee0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"session not created\",\"message\":\"session not created: Chrome failed to sta...wn>\\\\n#15 0x55f289d4f740 \\\\u003Cunknown>\\\\n#16 0x55f289d5f196 \\\\u003Cunknown>\\\\n#17 0x7f0a855b8609 start_thread\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\nE         (session not created: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x55f289d6031a <unknown>\nE       #1 0x55f2898766e0 <unknown>\nE       #2 0x55f2898addc8 <unknown>\nE       #3 0x55f2898a9926 <unknown>\nE       #4 0x55f2898f56f6 <unknown>\nE       #5 0x55f2898f4d46 <unknown>\nE       #6 0x55f2898e9203 <unknown>\nE       #7 0x55f2898b7cc0 <unknown>\nE       #8 0x55f2898b8c9e <unknown>\nE       #9 0x55f289d2dd0b <unknown>\nE       #10 0x55f289d31c92 <unknown>\nE       #11 0x55f289d1ab3c <unknown>\nE       #12 0x55f289d32807 <unknown>\nE       #13 0x55f289d000df <unknown>\nE       #14 0x55f289d4f578 <unknown>\nE       #15 0x55f289d4f740 <unknown>\nE       #16 0x55f289d5f196 <unknown>\nE       #17 0x7f0a855b8609 start_thread\n\n/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/errorhandler.py:229: SessionNotCreatedException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1732272399338,"stop":1732272399340,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1732272403152,"stop":1732272403290,"duration":138},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55f289d6031a <unknown>\n#1 0x55f2898766e0 <unknown>\n#2 0x55f2898addc8 <unknown>\n#3 0x55f2898a9926 <unknown>\n#4 0x55f2898f56f6 <unknown>\n#5 0x55f2898f4d46 <unknown>\n#6 0x55f2898e9203 <unknown>\n#7 0x55f2898b7cc0 <unknown>\n#8 0x55f2898b8c9e <unknown>\n#9 0x55f289d2dd0b <unknown>\n#10 0x55f289d31c92 <unknown>\n#11 0x55f289d1ab3c <unknown>\n#12 0x55f289d32807 <unknown>\n#13 0x55f289d000df <unknown>\n#14 0x55f289d4f578 <unknown>\n#15 0x55f289d4f740 <unknown>\n#16 0x55f289d5f196 <unknown>\n#17 0x7f0a855b8609 start_thread\n\n","statusTrace":"  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/usr/workspace/nintondo/autotests/conftest.py\", line 33, in driver\n    driver = webdriver.Chrome(options=options)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py\", line 212, in __init__\n    self.start_session(capabilities)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py\", line 299, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.8/dist-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"feature","value":"Test Design: Changing the password in the wallet"},{"name":"tag","value":"@pytest.mark.usefixtures('driver')"},{"name":"parentSuite","value":"tests.wallet_tests"},{"name":"suite","value":"test_wallet_change_password"},{"name":"host","value":"b3b395eb8b5c"},{"name":"thread","value":"9-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.wallet_tests.test_wallet_change_password"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"conf_pass","value":"')TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB'"},{"name":"expected_error","value":"'success'"},{"name":"new_pass","value":"')TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":5,"unknown":0,"total":7},"items":[{"uid":"f9527492778074ed","reportUrl":"https://Panda2k30.github.io/nintondo_autotests/42//#testresult/f9527492778074ed","status":"passed","time":{"start":1732271404892,"stop":1732271407037,"duration":2145}},{"uid":"9adc22c577e0ecb7","reportUrl":"https://Panda2k30.github.io/nintondo_autotests/41//#testresult/9adc22c577e0ecb7","status":"passed","time":{"start":1732114390817,"stop":1732114392999,"duration":2182}},{"uid":"fee720178257e232","reportUrl":"https://Panda2k30.github.io/nintondo_autotests/40//#testresult/fee720178257e232","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x560df365e31a <unknown>\n#1 0x560df31746e0 <unknown>\n#2 0x560df31abdc8 <unknown>\n#3 0x560df31a7926 <unknown>\n#4 0x560df31f36f6 <unknown>\n#5 0x560df31f2d46 <unknown>\n#6 0x560df31e7203 <unknown>\n#7 0x560df31b5cc0 <unknown>\n#8 0x560df31b6c9e <unknown>\n#9 0x560df362bd0b <unknown>\n#10 0x560df362fc92 <unknown>\n#11 0x560df3618b3c <unknown>\n#12 0x560df3630807 <unknown>\n#13 0x560df35fe0df <unknown>\n#14 0x560df364d578 <unknown>\n#15 0x560df364d740 <unknown>\n#16 0x560df365d196 <unknown>\n#17 0x7fd2cba3b609 start_thread","time":{"start":1732114015319,"stop":1732114015319,"duration":0}},{"uid":"f44563f56ae4b14d","reportUrl":"https://Panda2k30.github.io/Tests/39//#testresult/f44563f56ae4b14d","status":"passed","time":{"start":1732008760022,"stop":1732008762268,"duration":2246}},{"uid":"736ca58425b1e11","reportUrl":"https://Panda2k30.github.io/Tests/38//#testresult/736ca58425b1e11","status":"passed","time":{"start":1732006901036,"stop":1732006903195,"duration":2159}},{"uid":"804a9a1fbb14ed25","reportUrl":"https://Panda2k30.github.io/Tests/37//#testresult/804a9a1fbb14ed25","status":"passed","time":{"start":1732002722299,"stop":1732002724481,"duration":2182}}]},"tags":["@pytest.mark.usefixtures('driver')"]},"source":"13a7c03141831667.json","parameterValues":["')TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB'","'success'","')TFh1=!u5cBX9Q=E)WFO5*f7mJO7N&87D9vneR8bO69<(CfSf0Pp8s^lZX@p%3M9_smcB'"]}